recirq/qcqmc/hamiltonian.py

```diff
@@ -0,0 +1,319 @@
+from pathlib import Path
+from typing import Tuple, Union
+
+import attrs
+import fqe
+import h5py
+import numpy as np
+import openfermion as of
+from fqe.hamiltonians.restricted_hamiltonian import RestrictedHamiltonian
+from fqe.openfermion_utils import integrals_to_fqe_restricted
+from openfermionpyscf._run_pyscf import compute_integrals  # type: ignore
+from pyscf import ao2mo, fci, gto, scf
+
+from recirq.qcqmc.config import OUTDIRS
```

Comment:
Use import statements for packages and modules only, not for individual types, classes, or functions.

https://github.com/qh-lab/pyle/pull/new/u/dstrain/minor_uwaves_speed_up

Reply:
did you mean to include that link?

That's a google style guide thing, right? we don't follow that in ReCirq and just use pep8

Reply:
on a separate note would it be worthwhile (or possible) to add incremental formatting / lint checks for recirq?

Reply:
Oh it is linting with flake nvm